dockeradapter/dockeradapter.go

```diff
@@ -123,10 +124,23 @@ func ImageExists(ctx context.Context, client ImageInspector, imageName string) b
 func New() (Client, error) {
 	concreteClient := new(concreteDockerClient)
 	logging.Stderr("[Docker Adapter] Creating container runtime client: Docker")
+	ctx := context.Background()
 	dockerAPIClient, err := client.NewEnvClient()
 	if err != nil {
 		return nil, fmt.Errorf("[Docker Adapter] Unable to initialise container runtime type: Docker, error: %s", err)
 	}
+
+	serverVersion, _ := dockerAPIClient.ServerVersion(ctx)
+	clientVersion := dockerAPIClient.ClientVersion()
+	clientAPIVersion, _ := version.NewVersion(clientVersion)
+	serverAPIVersion, _ := version.NewVersion(serverVersion.APIVersion)
+
+	if clientAPIVersion.Compare(serverAPIVersion) == 1 {
+		return nil, fmt.Errorf(
+			"[Docker Adapter] Unable to initialize container runtime type: Docker, error: client is newer than server (client API version: %s, server API version: %s)",
```

Comment:
The above error message was presumably written by @johnmccabe "Unable to initialise container" on line 128/129 and this one by @jfryman "Unable to initialize container".

We should watch out for localisation issues like this, either by:

* standardising on z based on Puppet house style
* Not using words that exhibit the problem

I don't think it's worth expanding this to reviewing the whole codebase, but we should decide which approach we prefer and fix this particular file in this PR.